CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 ## [Unreleased]
 ### Added
 - Add metrics `cachecontroller_find_changes_and_invalidate_histogram` on latency for cache controller in finding changes and invalidating.  [#2135](https://github.com/openfga/openfga/pull/2135)
+- Improve `Check` performance when cache controller is enabled by invalidating asynchronously.  [#2124](https://github.com/openfga/openfga/pull/2124)
```

Comment:
```suggestion
- Improve `Check` performance when cache controller is enabled by invalidating _what?_ asynchronously.  [#2124](https://github.com/openfga/openfga/pull/2124)
```

Also, i just noticed that `cacheControllerEnabled` has no config flags, so how will users of openfga as a binary and not library use this?

Reply:
the cache controller enabled will be tracked via https://github.com/openfga/openfga/issues/2148